stdlib/src/socket.rs

```diff
@@ -80,14 +80,11 @@ mod _socket {
         vm.ctx.exceptions.os_error.clone()
     }
 
-    #[pyattr(once)]
+    #[pyattr(name = "timeout", once)]
     fn timeout(vm: &VirtualMachine) -> PyTypeRef {
-        vm.ctx.new_exception_type(
-            "socket",
-            "timeout",
-            Some(vec![vm.ctx.exceptions.os_error.clone()]),
-        )
+        PyType::new_simple_ref("TimeoutError", &error(vm)).unwrap()
```

Comment:
This is not OSError anymore after this change. Is this not an OSError?

Reply:
Oh yes. It is just TimeoutError which already exist in the RustPython.
https://github.com/python/cpython/blob/3.10/Modules/socketmodule.c#L7130
But actually this will not be act as I expected. I'll tell him to fix it 😊

Reply:
> This is not OSError anymore after this change. Is this not an OSError?

i modify